testsuite/tests/parsing/ocamltests

```diff
@@ -10,3 +10,4 @@ pr6604.ml
 pr6865.ml
 pr7165.ml
 shortcut_ext_attr.ml
+reloc.ml
```

Comment:
You're missing a trailing newline.

Reply:
Thanks, I've been struggling with that many times.
We should at least have proper error message in that case.

Reply:
`check-typo` outputs a warning in this case. If you don't have the check-typo hook in your git config, you should install it now.